packages/core/errors/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "@wallet-standard/errors",
+    "version": "1.0.0",
```

Comment:
I have no idea how versioning works in this repo. I can totally change it to whatever.

Reply:
I think it'll require creating a changeset to automatically publish things. So this could be set to 0.0.0 with a patch changeset provided to release 0.0.1.